blivet/deviceaction.py

```diff
@@ -581,7 +582,7 @@ def execute(self, callbacks=None):
 
         if isinstance(self.device.format, luks.LUKS):
             # LUKS needs to wait for random data entropy if it is too low
-            min_required_entropy = self.device.format.min_luks_entropy
+            min_required_entropy = luks_data.min_entropy
```

Comment:
I think this needs to check if ``format.min_luks_entropy`` is set or not and use the global value only if not.